### DISCUSSION TITLE
Player character renaming?

### DISCUSSION THREAD

**Discussion by Chonkblonk** [diasurgical/DevilutionX#6877] [Q&A] [answered]
Realized I maybe asked this in the wrong section, but is it possible to rename my player characters somehow?
I thought of a fun name for my Warrior, but then realized I could have done that name better, and now he's like level 15.

**Comment by AJenbo** [comment, marked as answer]
Didn't you ask this recently? There are no practical ways to do so, the easiest is probably creating a custom version that overwrites the player name when starting a game, but the technical skill required is pretty high

**Comment by Chonkblonk** [comment]
Ah, I figured as such. Oh well.
Yeah, I did, but not go reply, so I thought I maybe put it in the wrong section.

**Comment by qndel** [comment]
Personally I'm not a fan of such a feature because player names are the only thing that allows you to identify people - if you could easily change that, then nothing would stop people from renaming their character, ruining your game, then switching back and you'll never know who it was
